cmd/desync/config.go

```diff
@@ -64,6 +64,14 @@ func (c Config) GetS3CredentialsFor(u *url.URL) (*credentials.Credentials, strin
 		creds = NewStaticCredentials(credsConfig.AccessKey, credsConfig.SecretKey)
 	} else if credsConfig.AwsCredentialsFile != "" {
 		creds = NewRefreshableSharedCredentials(credsConfig.AwsCredentialsFile, credsConfig.AwsProfile, time.Now)
+	} else {
```

Comment:
I believe you could just change the above condition from
```go
} else if credsConfig.AwsCredentialsFile != "" {
```

to:
```go
} else {

That that would have the same effect, without the extra `else`, wouldn't it?
```

Reply:
Good spot, I just pushed that change